_layouts/2017_home.html

```diff
@@ -17,11 +17,11 @@
                 <div class="intro-lead-in">An asynchronous-first virtual conference for the Bayesian community</div>
                 <div class="intro-lead-in-sub">First Week of November (Tentative)</div>
                 <div class="intro-lead-in-sub">Location: Global</div>
-                <a href="./cfp" class="btn btn-info btn-lg btn-mr-10">Give a talk</a>
+                <a href="./cfp" class="btn btn-info btn-lg btn-margins">Talk at PyMCon</a>
```

Comment:
Can this line be changed to something like  "Submit a proposal"? Talks are just one of the formats, we have tutorials and "Build a Model" as well!

Reply:
Done